### PR TITLE
Correction to the MEA solvent property package and column

### PR DESCRIPTION
**Pull request by anujad95**
## Fixes
MEA_solvent property package and solvent_column unit model

## Summary/Motivation:
Corrections to the molar enthalpy calculation and MEA enthalpy of vaporization in the MEA_solvent property package, along with the enthalpy transfer term in solvent_column unit model

## Changes proposed in this PR:
- Converting the temperature used in the molar enthalpy calculation for liquid phase components, from K to deg C
- Including the value for MEA enthalpy of vaporization to ensure accurate molar enthalpy calculation
- Correcting the enthalpy transfer term in solvent_column unit model

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: idaes/models_extra/column_models/properties/MEA_solvent.py [hunk 1]
```diff
@@ -910,7 +910,7 @@ def calculate_scaling_factors(b, rblock):
                     "2": (-4.51417e-4, pyunits.g / pyunits.mL / pyunits.K),
                     "3": (1.19451, pyunits.g / pyunits.mL),
                 },
-                "dh_vap": 0,  # MEA is assumed to be non-volatile
+                "dh_vap": 54000,
```
Comment (andrewlee94) [inline idaes/models_extra/column_models/properties/MEA_solvent.py:913]: Is this actually used anywhere?

**Comment by ksbeattie** [comment]
@anujad95 is this ready for a re-review?

**Comment by andrewlee94** [comment]
@anujad95 There are a lot of failing tests in this PR which need to be fixed before we can review it. You should also update the branch as well.

**Comment by ksbeattie** [comment]
I've updated this branch from main in hopes that this can be merge in time for the Aug release (meaning next week).

File: idaes/models_extra/column_models/tests/test_solvent_condenser.py [hunk 2]
```diff
@@ -300,6 +301,9 @@ def model(self):
 
         m.fs.unit.heat_duty.fix(-6264)
 
+        xfrm = TransformationFactory("contrib.strip_var_bounds")
```
Comment (andrewlee94) [inline idaes/models_extra/column_models/tests/test_solvent_condenser.py:304]: Could you comment on why this is needed? An inline comment in the code might also be good for future reference.

File: idaes/models_extra/column_models/tests/test_plate_heat_exchanger.py [hunk 3]
```diff
@@ -211,10 +211,10 @@ def test_solution(self, phe):
 
         assert pytest.approx(244327, rel=1e-5) == value(phe.fs.unit.heat_duty[0])
 
-        assert pytest.approx(365.288, rel=1e-5) == value(
+        assert pytest.approx(345.612, rel=1e-5) == value(
```
Comment (andrewlee94) [inline idaes/models_extra/column_models/tests/test_plate_heat_exchanger.py:214]: These are some significant changes in the model results - have you confirmed that you have verified these changes are correct?

**Review by andrewlee94 — CHANGES_REQUESTED** [review]
This mostly looks good but I would like to confirm that the changes in model results we are seeing for the heat exchanger models have been verified against some other source.

**Review by andrewlee94 — APPROVED** [review]

**Review by aostace01 — APPROVED** [review]
Looks good to me.

**Pull request merged**